README.md

```diff
@@ -63,6 +63,14 @@ my-microservice/
   package.json
 ```
 
+It is recommended that you set the `CI` environment variable when `npm install`ing in CI so that you do not attempt to double-install files.
```

Comment:
Can you put this comment below [this paragraph](https://github.com/mixmaxhq/install-files/pull/10/files#diff-04c6e90faac2675aa89e2176d2eec7d8R85) please? Also say that it's because CI may be a development environment (`npm install`, no `--production`), but you still don't want to run `install-files` there.